app/views/dashboards/_dashboard_article.html.erb

```diff
@@ -49,7 +49,7 @@
       <% end %>
     <% elsif article.published %>
       <a href="<%= article.path %>/manage" class="cta pill black">MANAGE</a>
-    <% else %>
+    <% elsif @user == article.user %>
```

Comment:
This logic would work in the context of the org admin, but not necessarily DEV admins because we set `@user` to the user we're looking to administrate and then `@user == article.user` would always be true, eh?

I don't think this is too big a deal because I figure the more important case here is org admins.

Reply:
Oh hmm, good point. 🤔 If I think of a workaround then I'll update the logic there.